src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj

```diff
@@ -302,5 +302,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Fhir.Api\Microsoft.Health.Fhir.Api.csproj" />
     <ProjectReference Include="..\Microsoft.Health.Fhir.Core\Microsoft.Health.Fhir.Core.csproj" />
+    <ProjectReference Include="..\Microsoft.Health.Fhir.CosmosDb\Microsoft.Health.Fhir.CosmosDb.csproj" />
```

Comment:
Whats the dependency here?

Reply:
That project has two constants defined for the storage emulator host name and key. I thought about defining them locally in EnvironmentVariables class, but wanted to check how everyone thinks about adding this dependency. Please let me know what you think.

Reply:
I'd prefer not adding the dependency and copying the values over locally. I think in general common packages shouldn't rely on specific packages (like for datastore). It should be the other way around.

Reply:
Fixed as suggested.